### ISSUE TITLE
Fix project service to get new projects (with details JSON) in ES cluster

### ISSUE THREAD

**Issue by vikasrohit** [appirio-tech/connect-app#894]
As of now, the project service fails to create a new project in ES cluster if the payload contains the details JSON field which is preventing the new projects to be listed on the listing page.

**Comment by vikasrohit** [comment]
It seems ES does not allow indexing same field with two different types. Since we made number of screens and primary target questions non skippable in specification tab, it started to fail updating the project update in ES cluster because of that reason. When these were skippable questions, the value of corresponding fields was an object and since we made it compulsory field, it started to save a primitive value in it which is causing ES cluster to fail in indexing the updates. The same error is causing troubles with indexing the project on creation because now with new flow we are setting the values for these two fields (along with goal and users questions) as concrete/primitive value instead of an object. This was preventing the new challenges from listing on projects page.

**Comment by vikasrohit** [comment]
After more debugging found that production is working fine because we never had, on production, the `numberScreens` and `primaryTarget` questions as optional fields in project specification form. In dev initially they were optional but before going live we updated them to be non skippable fields, hence dev contains both type of data.
Fixed the bad data on dev ES cluster by deleting the index and recreating it with data from postgres and handling the bad data in recreation script (`npm run migrate:es`). Data in postgres is  still bad, so we might to fix them as well. However, not doing that right away, instead deleting the projects with bad data from dev.
Also, reverted previous changes for fixing the same issue and now making the only required changes for new projects.

fyi @fnisen (nothing to worry now, but I was worried if we need to resync the data on production as well, however, it seems production is good, still fingers crossed for the deployment day 😬 .)
